src/main/kotlin/lotto/LottoNumberValidator.kt

```diff
@@ -0,0 +1,22 @@
+package lotto
+
+object LottoNumberValidator {
+    const val LOTTO_MIN_RANGE = 1
+    const val LOTTO_MAX_RANGE = 45
```

Comment:
소소하지만 range를 상수값으로 정의할수 있을거같아요 :)
```suggestion
    const val LOTTO_RANGE = 1..45
```